addons/mail/views/assets.xml

```diff
@@ -159,6 +159,9 @@
                 <script type="text/javascript" src="/mail/static/src/widgets/discuss_invite_partner_dialog/discuss_invite_partner_dialog.js"></script>
                 <script type="text/javascript" src="/mail/static/src/widgets/form_renderer/form_renderer.js"></script>
                 <script type="text/javascript" src="/mail/static/src/widgets/messaging_menu/messaging_menu.js"></script>
+                <script type="text/javascript" src="/mail/static/src/components/notification_alert/notification_alert.js"></script>
+                <script type="text/javascript" src="/mail/static/src/components/notification_request/notification_request.js"></script>
+                <script type="text/javascript" src="/mail/static/src/widgets/notification_alert/notification_alert.js"></script>
```

Comment:
alphabetical order (by each path bit)

Reply:
Look like it's already the case...

Reply:
not with other paths already in bundle

Reply:
For instance, for components:
- ...
- **<script type="text/javascript" src="/mail/static/src/components/notification_alert/notification_alert.js"></script>**
- <script type="text/javascript" src="/mail/static/src/components/notification_group/notification_group.js"></script>
- <script type="text/javascript" src="/mail/static/src/components/notification_list/notification_list.js"></script>
- <script type="text/javascript" src="/mail/static/src/components/notification_popover/notification_popover.js">- </script>
- **<script type="text/javascript" src="/mail/static/src/components/notification_request/notification_request.js"></script>**
- ...